.github/workflows/weblate-sync-pot.yml

```diff
@@ -16,6 +16,12 @@ jobs:
       options: --user root
     timeout-minutes: 10
     steps:
+      - uses: actions/create-github-app-token@v1
+        id: app-token
+        with:
+          app-id: ${{ vars.RSM_CI_APP_ID }}
+          private-key: ${{ secrets.RSM_CI_APP_PRIVATE_KEY }}
+
```

Comment:
Since the token is used to create a commit in the weblate repository (dnf5-l10n), shouldn't we create token with
```
permissions: '{"contents":"write"}'
```

Reply:
Also we might need to specify the repository to grant access to. According to documentation if both owner and repositories are empty, access will be scoped to only the current repository.
```
repositories: '["rpm-software-management/dnf5-l10n"]'
```